packages/create-stylable-app/template/ts-react-webpack/template.js

```diff
@@ -38,4 +40,5 @@ module.exports = {
             test: 'npm run typecheck && npm run lint',
         },
     },
+    postinstall: [['npm', 'run', 'build']],
```

Comment:
`postinstall` runs when installing an npm package as a dep. if anything, use `prepare`, but I'm not sure this is even a wanted behavior...

Reply:
This is our "postinstall", it's not added as a script to the user's package json